api/v2/serializers/details/__init__.py

```diff
@@ -10,12 +10,42 @@
 from .project import ProjectSerializer
 from .project_instance import ProjectInstanceSerializer
 from .project_volume import ProjectVolumeSerializer
-from .provider import ProviderSerializer, ProviderTypeSerializer, PlatformTypeSerializer
+from .provider import ProviderSerializer, ProviderTypeSerializer,\
+    PlatformTypeSerializer
 from .provider_machine import ProviderMachineSerializer
 from .quota import QuotaSerializer
-from .resource_request import ResourceRequestSerializer, UserResourceRequestSerializer
+from .resource_request import ResourceRequestSerializer,\
+    UserResourceRequestSerializer
 from .size import SizeSerializer
 from .status_type import StatusTypeSerializer
 from .tag import TagSerializer
 from .user import UserSerializer
-from .volume import VolumeSerializer
+from .volume import VolumeSerializer, UpdateVolumeSerializer
+
+__all__ = (
```

Comment:
The purpose of `__init__.py` is for creating a package. The variable `__all__` restricts what is imported when importing the entire package. See https://docs.python.org/2/tutorial/modules.html#importing-from-a-package

The second reason is that we import everything and never use it which will fail flake8 test.

I decided to add it because it doesn't take too much extra work to keep the namespace from being polluted if everything was imported.